content/blueprints/gcp.md

```diff
@@ -0,0 +1,1277 @@
+    Optional instance name. By default it will be instance id.
+
+    *type:* string
+    *default:*
+  * `zone`
+    Optional zone name. If not given, this instance will be deployed in default zone.
+
+    *type:* string
+    *default:*
+  * `tags`
+    Optional tags. If not given, this instance will have a tag only with its name.
+
+    *type:* string
+    *default:*
+  * `external_ip`
+    Should the Instance be created with an externally-accessible IP address. This will be an ephemeral IP. If you would like to use an IP address which can be transferred to another Instance then connect this Instance to an `Address` node using the `cloudify.gcp.relationships.instance_connected_to_ip` relationship.
+
+    *type:* boolean
+    *default:* False
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+  * `block_project_ssh_keys`
+    Disable project-wide ssh keys for this Instance
+
+    *type:* boolean
+    *default:* False
+  * `image_id`
+    The ID of the image in your GCP account.
+
+    *type:* string
+    *default:* {}
+  * `additional_settings`
+    Additional instance settings.
+
+    *default:* {}
+  * `startup_script`
+    A script which will be run when the Instance is first started example:
+      type: string
+      script: |
+        yum install some stuff
+        systemctl start it
+    or:
+      type: file
+      script: <path to script file>
+
+    *default:*
+  * `can_ip_forward`
+    Is the VM allowed to send packets with source address different to its own?
+
+    *type:* boolean
+    *default:* False
+  * `use_external_resource`
+    Indicate whether the resource exists and use existing (true)  or if Cloudify should create new resource (false).
+
+    *type:* boolean
+    *default:* False
+
+
+
+### Example
+
+{{< gsHighlight  yaml  >}}
+
+my_gcp_instance:
+  type: cloudify.gcp.nodes.Instance
+  properties:
+    image_id: http://url.to.your.example.com/image
+    instance_type: n1-standard-1
+    gcp_config:
+      project: your-project
+      network: default
+      zone: us-east1-b
+      auth: path_to_auth_file.json
+
+{{< /gsHighlight >}}
+
+This example includes shows adding additional parameters, tagging an instance name, and explicitly defining the gcp_config.
+
+
+
+
+## cloudify.gcp.nodes.InstanceGroup
+**Derived From:** [cloudify.nodes.Root]({{< relref "blueprints/built-in-types.md" >}})
+
+A GCP InstanceGroup.
+This is used to configure failover systems. InstanceGroups can be configured to scale automatically based on load, and will replace failing Instances with freashly started ones.
+
+
+**Properties:**
+
+
+  * `gcp_config`
+    A dictionary of values to pass to authenticate with the GCP API.
+
+    *default:* {}
+  * `additional_settings`
+    Additional setting for instance group
+
+    *default:* {}
+  * `use_external_resource`
+    Indicate whether the resource exists and use existing (true) or if Cloudify should create new resource (false).
```

Comment:
Indicate**s** whether the resource exists **and should be used (true),** or if Cloudify should create the resource **(false).**